src/main/resources/wa-task-completion-civil-civil.dmn

```diff
@@ -188,18 +188,6 @@
           <text>"Auto"</text>
         </outputEntry>
       </rule>
-      <rule id="DecisionRule_05wason">
```

Comment:
I have removed this for now to avoid potential conflicts with the existing "TAKE_CASE_OFFLINE" version of transferCaseOffline that is used by prod.